packages/libsql-client/src/lib/libsql-js.ts

```diff
@@ -77,6 +78,12 @@ export function connect(config: Config): Connection {
     if (url.protocol == "http:" || url.protocol == "https:") {
         return new Connection(new HttpDriver(url));
     } else {
-        return new Connection(new SqliteDriver(rawUrl));
+        // Hack taken from https://stackoverflow.com/a/31090240/1474847
+        const isBrowser = new Function("try {return this===window;}catch(e){ return false;}");
```

Comment:
There are some other hacks that will go in the other direction, creating and populating a `window` in order to  polyfill web APIs on node. This would seem incompatible with those.

Maybe the browser-or-node package recommended as a different answer would be better?

Reply:
How about: https://stackoverflow.com/a/31456668/807126